src/DataSource/Types.hs

```diff
@@ -347,6 +350,10 @@ newtype CommentOuter = CommentOuter {
       coComment :: Comment
     }
 
+newtype LogFiles = LogFiles {
```

Comment:
I prefer the formatting seen here - https://github.com/input-output-hk/log-classifier/pull/26/files#diff-ee661ebeb72d3f09577f0fd782917c5aR359 since it doesn't change any other lines besides the ones you change which is helpful.